scripts/alert_emails/template.html

```diff
@@ -10,673 +10,7 @@
 <!--[if !mso]><!--><meta http-equiv=" X-UA-Compatible" content="IE=Dedge">
     <!--<![endif]-->
     <meta name="viewport" content="width=device-width">
```

Comment:
I had to remove the `<style>` blocks since they don't really work in email and we no longer can use CampaignMonitor's "InclineCSS" feature to inline them.  I fixed it myself by just taking the result of one of the Campaign Monitor emails and pasting it back into template.html (and then putting the handlebars template things back in and removing the CM click tracking stuff).  Big hack but seems to have worked. 🤷‍♂️